components/org.wso2.carbon.identity.oauth.dcr.endpoint/src/main/resources/api.identity.oauth.dcr.endpoint.yaml

```diff
@@ -300,6 +300,8 @@ definitions:
           type: string
       ext_param_sp_template:
         type: string
+      ext_saas_app:
```

Comment:
Can we set the default value here to false?

Reply:
Fixed